graylog2-server/src/main/java/org/graylog2/shared/rest/CORSFilter.java

```diff
@@ -29,7 +29,11 @@
     public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) throws IOException {
         // we have already added the necessary headers for OPTIONS requests below
         if (requestContext.getRequest().getMethod().equalsIgnoreCase("options")) {
-            return;
+            if(Response.Status.Family.familyOf(responseContext.getStatus()) == Response.Status.Family.INFORMATIONAL) {
+                return;
+            }
+            responseContext.setStatus(Response.Status.NO_CONTENT.getStatusCode());
+            responseContext.setEntity("");
         }
```

Comment:
Ah, that makes sense. I was wondering why checking informational responses :) 
The requests I was talking about were 20xs, can't remember if it also happened with other responses but will double check now.